Create ui-c8y-1022-28-0-show-all-children-of-a-group-including-grand-children-on-maps.md (#3709)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-28-0-show-all-children-of-a-group-including-grand-children-on-maps.md

---
date: ""

> title: Group children and grandchildren are now shown on maps

Reviewer: ```suggestion
Maps now display devices across all nested group levels
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64602
version: 1022.28.0
---

> Previously, only direct child devices of a group were shown on a map when that group was selected. To provide a more complete overview, the map now shows all descendants of a selected group, including both direct children and grandchildren. This change allows users to see the full device hierarchy and geographic distribution for a group on the map without needing to manually drill down into each level of subgroups.

Reviewer: ```suggestion
Previously, only direct child devices of a group were displayed on a map when that group was selected. Now, the map shows all devices within the group hierarchy, including those in nested subgroups. This change provides a complete view of the device hierarchy and geographic distribution for a group, without the need to manually drill down into each nested level.
```
